src/main/java/mate/academy/spring/service/UserServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.service;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Autowired
+    private UserDao userDao;
```

Comment:
```suggestion
    private final UserDao userDao;
```
краще для залеженостой всюду юзати однаковий підхід для консистенції як ти зробив з sessionFactory на дао рівні